infrastructure/backup/README.md

```diff
@@ -0,0 +1,70 @@
+
+### Deployment
+
+```bash
+helm repo add vmware-tanzu https://vmware-tanzu.github.io/helm-charts
+helm install velero vmware-tanzu/velero --namespace velero --create-namespace -f values.yaml --dry-run
+```
+
+[](https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/README.md)
+
+## Usage
+
+It's better to use Velero's client in order to interact with its deployment on the cluster. On [the official documentation](https://velero.io/docs/) it's possible to see how to install the client, in particular [here](https://velero.io/docs/v1.6/basic-install/#install-the-cli) are the instructions for the currently installed version.
+
+### Prerequisites: volumes preparation
+
+The volume snapshotting of Velero through Restic can work on a whitelist or blacklist basis. In the case of CrownLabs it has been chosen to use the whitelist approach since crucial volumes to be persisted can be identified. In order to tell Velero to backup a volume it's necessary to add a special annotation to the pods to which such volumes are mounted: `backup.velero.io/backup-volumes=VOL_NAME_2_,VOL_NAME_2,...`. 
+
+### Backup schedule
+
+To create a backup schedule it's sufficient to run 
+```bash
+velero schedule create <SCHEDULE_NAME> --schedule="0 2 * * *"
+```
+The previous example would trigger a backup every day at 2 AM, for every resource found in every namespace. 
+- It's possible to specify an a time to live for created backups through the `--ttl` switch (that accepts a string like `12h34m56s`) after which the backup and associated data would be removed from the backup location. It defaults to 30 days. 
+- It's possible to specify which namespaces have to be backed up through the switch `--include-namespaces` (comma separated). In case this isn't specified, any namespace will be included in the backup. 
+
+### Single backup
+
+It's possible to trigger backups using `velero backup create <BACKUP_NAME>` togerther with the other switches and/or in combination with `--from-schedule <SCHEDULE_NAME>` to inherit options from the specified schedule.
+
+### Restore
+
+It's possible to make Velero restore a backup through the option `velero restore create <BACKUP_NAME>`. Options are available to perform a partial restore.
+
+### Monitoring
+
+The previous commands will request Velero to perform respective operations but the command would return immediately, effectively just creating the corresponding Velero resource.
+To monitor the progress of the requested operation it's necessary to run `velero backup get` or `velero restore get` to obtain information about running operations and the corresponding `describe` (instead of `get`), available at the end of each process, to see the results. 
```

Comment:
```suggestion
To monitor the progress of the requested operation it is necessary to run `velero backup get` or `velero restore get` to obtain information about running operations and the corresponding `describe` command (instead of `get`), available at the end of each process, to see the results. 
```